projects/igniteui-angular/src/lib/core/styles/components/overlay/_overlay-theme.scss

```diff
@@ -66,7 +66,7 @@
         background: transparent;
         transition: background .25s $ease-in-out-quad;
         pointer-events: none;
-        z-index: 10;
+        z-index: 10005;
```

Comment:
@wnvko Do you foresee any issues arsing from this change?

Reply:
nope, from overlay point of view.
However, if one has set `z-index` in application in order to show something above the overlay, should now update the `z-index`. However this looks like very rare application scenario - why one should show something above the overlay :)